CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to ProtoSchool
+
+### What is ProtoSchool?
+
+ProtoSchool is an educational community dedicated to making it easy to get started with decentralized technologies. It's an initiative of [Protocol Labs](https://protocol.ai/), which sponsors such open source projects as [IPFS](https://ipfs.io/), [IPLD](https://ipld.io/), [libp2p](https://libp2p.io/), and [Multiformats](https://multiformats.io/). However, we depend on volunteers to develop our coding workshops and make our content available at chapters around the world through live events. We always need more volunteers, and there are always things to do!
+
+ProtoSchool has two main initiatives:
+
+- Building and maintaining workshops that teach about the protocols underlying the decentralized web, hosted at [https://proto.school/#/](https://proto.school/#/)
+- Organizing educational events around the world
+
+As a community, we support the following principles:
+
+- Beginners should feel welcome
+- Chapters should experiment with new ways to engage their community, and share approaches with other chapters over time
+- Events should be affordable and should strive towards diversity and inclusiveness
+- We are 100% open source and like to license things as open source/creative commons
+
+### Where does the name come from?
+
+The "Proto" in ProtoSchool represents the networking protocols that support the decentralized web, and is a shout-out to [Protocol Labs](https://protocol.ai/), the organization that sponsors our work.
+
+We also take a _lot_ of inspiration from [NodeSchool](https://nodeschool.io/). (Imitation is the sincerest form of flattery, and all that jazz.)
+
+### Ways to contribute
+
+#### Join (or start) a local chapter
+
+See if there is a [chapter near you](http://proto.school/chapters.html). Each chapter gets their own github repo, e.g. https://github.com/protoschool/san-francisco. These repos usually serve at least two purposes (maybe more):
+
+- host a chapter page on the `gh-pages` branch ([example](https://github.com/protoschool/san-francisco/tree/gh-pages))
+- provice a discussion board for the chapter using GitHub Issues ([example](https://github.com/protoschool/san-francisco/issues))
+
+There might be an issue for new or prospective members to introduce themselves, or there may be one that is an announcement/planning for the next event. To receive emails when new discussions happen, be sure to click "Watch" on your chapter's repo.
+
+If there is no local chapter near you and you'd like to start one, [check out our guide](https://github.com/protoschool/organizing/#how-to-start-a-new-protoschool-chapter) for instructions on how to get started.
+
+#### Help answer questions and provide tech support
+All of our workshops point people at this issue board when they get stuck:
+
+[https://github.com/protoschool/protoschool.github.io/issues](https://github.com/ipfs-shipyard/proto.school/issues)
```

Comment:
Should we create a new repo called `help` to handle issues where people get stuck in order to separate them from actual code issues in the site or workshop?

Reply:
Let's leave it for the moment and re-assess when we have more traffic. Many beginners won't know whether the problem is the lesson itself (poor instructions, bad test code) or them being individually confused, so I don't want them to have to guess at which is true and I don't want us to lose track of feedback we should be using to make the site more clear.